### DISCUSSION TITLE
Unable to login to my account

### DISCUSSION THREAD

**Discussion by completecommunication** [community/community#142375] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Good day, I have been unable to login to my account please simplify a simple login for me it kept telling me wait a moment

**Comment by LiteBrite82** [comment]
Hi @completecommunication ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by completecommunication** [reply]
O thanks
